GITHUB_ORG_MANAGEMENT_POLICY.md

```diff
@@ -1,82 +1,79 @@
 # Node.js GitHub Organization Management Policy
 
-The Node.js GitHub Organizations:
+The following GitHub Organizations are development resources under the direction
+of the Node.js Technical Steering Committee (TSC):
 
-* (https://github.com/nodejs)
-* (https://github.com/pkgjs)
-
-are development resources under the direction
-of the Node.js Technical Steering Committee (TSC)
+* [`nodejs`](https://github.com/nodejs)
+* [`pkgs`](https://github.com/pkgjs)
 
 ## Node.js Admin Repository
 
 The [Node.js admin repository][nodejs/admin] serves as the
-central location for managing Node.js GitHub Organization administrative
-activities. Only Node.js GitHub Organization owners and TSC members have write permissions to 
-the Node.js admin repository.
+central location for managing our GitHub Organization administrative
+activities. This repository is maintained by the TSC.
 
 ## Organization Roles
 
 ### Owners
 
-Whether to grant Owner permissions is determined by optimizing
+Whether to grant Owner role is determined by optimizing
 for the following conflicting requirements:
 
-* Limiting access to reduce risk
-* Enabling individuals to move community work forward without undue delay
+* Limiting access to reduce risk.
+* Enabling individuals to move community work forward without undue delay.
 
 When possible, automation and tools should be used to reduce the breadth of
 access that needs to be provided in order to enable individuals to move
 community work forward. As these tools are created, the groups to which
-Owner permissions are granted will be reduced.
+Owner role are granted will be reduced.
 
-The following groups are granted Ownership permissions:
+The following groups have the role "Owner" in our GitHub Organizations:
 
-* **TSC members.**
+* **TSC voting members**.
 * **OpenJS Director of Program Management**. The OpenJS Director of Program
   Management will limit their use of the access granted to accept GitHub terms
   and conditions. They will be added to the team called
   [openjs-administrative-support](https://github.com/orgs/nodejs/teams/openjs-administrative-support)
```

Comment:
Ah, this team doesn't exist, assuming it did at some point and I just recreated one yesterday as Ben was in 0 teams I wonder what happened:

```suggestion
  [openjs](https://github.com/orgs/nodejs/teams/openjs)
```